src/hw9/ArraySizeException.java

```diff
@@ -1,6 +1,6 @@
 package hw9;
 
-public class ArraySizeException extends Throwable{
-    public ArraySizeException ( String s ){
+public class ArraySizeException extends Throwable {
+    public ArraySizeException (String s) {
```

Comment:
If you're overriding a constructor and it accepts a parameter, then the constructor's body should have this line `super(s)`, please add it.